var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -54,13 +42,10 @@ class Flecsi(CMakePackage, CudaPackage):
         description="Backend to use for distributed memory",
         multi=False,
     )
-    variant("debug_backend", default=False, description="Build Backend with Debug Mode")
-    variant("disable_metis", default=False, description="Disable FindPackageMetis")
     variant("shared", default=True, description="Build shared libraries")
     variant("flog", default=False, description="Enable flog testing")
-    variant("doxygen", default=False, description="Enable doxygen")
-    variant("doc", default=False, description="Enable documentation")
-    variant("coverage", default=False, description="Enable coverage build")
+    variant("graphviz", default=False, description="Enable GraphViz Support")
+    variant("doc", default=False, description="Enable documentation", when="@:1.9,2.2:")
```

Comment:
Wouldn't `@:2,2.2:` work just as well without pretending that 9 is &infin;?  (Similarly below.)

Reply:
Actually, looking backwards, the `doc` variants should also work with `2.0.0` and `2.1.0`, so I removed the extra condition.

Reply:
`1.9` is used as a boundary in various other places.

Reply:
Yes, and that is correct. It's an inclusive boundary, which would be wrong with `:2`.

Reply:
Even inclusive, there's no actual version `2` here, so `2.0.0` would be outside the range, right?

Reply:
Rereading the documentation:
> Version ranges can be open, e.g. `:3` means any version up to and including `3`. This would include `3.4` and `3.4.2`.

So it should be `@:1` (in appropriate places that still exist).

Reply:
Done.

Reply:
There are `1.4.99`s as well that could presumably be `1.4`.

Reply:
Done, again.